src/libraries/System.IO.Hashing/src/System/IO/Hashing/XxHashShared.cs

```diff
@@ -0,0 +1,785 @@
+        };
+
+        // Cast of DefaultSecret byte[] => ulong[] (See above for the correspondence)
+        public const ulong DefaultSecretUInt64_0 = 0xBE4BA423396CFEB8;
+        public const ulong DefaultSecretUInt64_1 = 0x1CAD21F72C81017C;
+        public const ulong DefaultSecretUInt64_2 = 0xDB979083E96DD4DE;
+        public const ulong DefaultSecretUInt64_3 = 0x1F67B3B7A4A44072;
+        public const ulong DefaultSecretUInt64_4 = 0x78E5C0CC4EE679CB;
+        public const ulong DefaultSecretUInt64_5 = 0x2172FFCC7DD05A82;
+        public const ulong DefaultSecretUInt64_6 = 0x8E2443F7744608B8;
+        public const ulong DefaultSecretUInt64_7 = 0x4C263A81E69035E0;
+        public const ulong DefaultSecretUInt64_8 = 0xCB00C391BB52283C;
+        public const ulong DefaultSecretUInt64_9 = 0xA32E531B8B65D088;
+        public const ulong DefaultSecretUInt64_10 = 0x4EF90DA297486471;
+        public const ulong DefaultSecretUInt64_11 = 0xD8ACDEA946EF1938;
+        public const ulong DefaultSecretUInt64_12 = 0x3F349CE33F76FAA8;
+        public const ulong DefaultSecretUInt64_13 = 0x1D4F0BC7C7BBDCF9;
+        public const ulong DefaultSecretUInt64_14 = 0x3159B4CD4BE0518A;
+        public const ulong DefaultSecretUInt64_15 = 0x647378D9C97E9FC8;
+
+        // Cast of DefaultSecret offset by 3 bytes, byte[] => ulong[]
+        public const ulong DefaultSecret3UInt64_0 = 0x81017CBE4BA42339;
+        public const ulong DefaultSecret3UInt64_1 = 0x6DD4DE1CAD21F72C;
+        public const ulong DefaultSecret3UInt64_2 = 0xA44072DB979083E9;
+        public const ulong DefaultSecret3UInt64_3 = 0xE679CB1F67B3B7A4;
+        public const ulong DefaultSecret3UInt64_4 = 0xD05A8278E5C0CC4E;
+        public const ulong DefaultSecret3UInt64_5 = 0x4608B82172FFCC7D;
+        public const ulong DefaultSecret3UInt64_6 = 0x9035E08E2443F774;
+        public const ulong DefaultSecret3UInt64_7 = 0x52283C4C263A81E6;
+        public const ulong DefaultSecret3UInt64_8 = 0x65D088CB00C391BB;
+        public const ulong DefaultSecret3UInt64_9 = 0x486471A32E531B8B;
+        public const ulong DefaultSecret3UInt64_10 = 0xEF19384EF90DA297;
+        public const ulong DefaultSecret3UInt64_11 = 0x76FAA8D8ACDEA946;
+        public const ulong DefaultSecret3UInt64_12 = 0xBBDCF93F349CE33F;
+        public const ulong DefaultSecret3UInt64_13 = 0xE0518A1D4F0BC7C7;
+
+        public const ulong Prime64_1 = 0x9E3779B185EBCA87UL;
+        public const ulong Prime64_2 = 0xC2B2AE3D27D4EB4FUL;
+        public const ulong Prime64_3 = 0x165667B19E3779F9UL;
+        public const ulong Prime64_4 = 0x85EBCA77C2B2AE63UL;
+        public const ulong Prime64_5 = 0x27D4EB2F165667C5UL;
+
+        public const uint Prime32_1 = 0x9E3779B1U;
+        public const uint Prime32_2 = 0x85EBCA77U;
+        public const uint Prime32_3 = 0xC2B2AE3DU;
+        public const uint Prime32_4 = 0x27D4EB2FU;
+        public const uint Prime32_5 = 0x165667B1U;
```

Comment:
These now duplicated with https://github.com/dotnet/runtime/blob/a4e98d40a098a0a58a88c1205749050cfb5f9881/src/libraries/System.IO.Hashing/src/System/IO/Hashing/XxHash64.State.cs#L16-L20 and https://github.com/dotnet/runtime/blob/a4e98d40a098a0a58a88c1205749050cfb5f9881/src/libraries/System.IO.Hashing/src/System/IO/Hashing/XxHash32.State.cs#L16-L20.  Can we dedup them?

Reply:
Fixed by commit b0fe87695a858d5a8ca02b19d17eb18f2e486884